tests/test_usage.py

```diff
@@ -379,6 +381,28 @@ def test_hear_works_WHEN_fuzzed_and_say_works(self):
 			theResult = False
 		self.assertTrue(theResult, fail_fixture)
 
+	def test_say_works_WHEN_using_stdin(self):
```

Comment:
@coderabbitai, is it possible to group tests by category or similar meta-data and then selectively invoke unittest to only selectively run some of those tests? Is there another way to achieve similar functionality?